packages/app/components/Queue/TA/TAQueueList.tsx

```diff
@@ -259,16 +262,46 @@ const NoQuestionsText = styled.div`
   color: #212934;
 `;
 
+function NotifReminderButton() {
+  const [isNotifOpen, setIsNotifOpen] = useState(false);
```

Comment:
probably would be better named something like "signedUpForNotifications" or "hasSignedUpForNotifications" or something -- isNotifOpen is a little misleading, it implies that there's a notificaiton open on the screen